Doc/whatsnew/2.7.rst

```diff
@@ -1540,11 +1540,11 @@ changes, or look through the Subversion logs for all the details.
 * The :mod:`ssl` module's :class:`~ssl.SSLSocket` objects now support the
   buffer API, which fixed a test suite failure (fix by Antoine Pitrou;
   :issue:`7133`) and automatically set
-  OpenSSL's :c:macro:`SSL_MODE_AUTO_RETRY`, which will prevent an error
+  OpenSSL's :c:macro:`!SSL_MODE_AUTO_RETRY`, which will prevent an error
   code being returned from :meth:`recv` operations that trigger an SSL
   renegotiation (fix by Antoine Pitrou; :issue:`8222`).
 
-  The :func:`ssl.wrap_socket` constructor function now takes a
+  The :func:`!ssl.SSLContext.wrap_socket` constructor function now takes a
```

Comment:
Why suppress the link here with `!`? It's documented here, no? https://docs.python.org/3/library/ssl.html#ssl.SSLContext.wrap_socket

Reply:
Indeed it is! I probably meant `~` to keep the display as before?

```suggestion
  The :func:`~ssl.SSLContext.wrap_socket` constructor function now takes a
```